azure-pipelines/templates/package.yml

```diff
@@ -7,22 +7,33 @@ steps:
       workingDir: $(working_directory)
 
   - task: CopyFiles@2
-    displayName: "Copy package to staging directory"
+    displayName: "Copy packages and vsix to staging directory"
     inputs:
       Contents: |
         **/*.vsix
         **/*.tar.gz
         **/*.tgz
-      TargetFolder: "$(build.artifactstagingdirectory)"
+      TargetFolder: "$(build.artifactstagingdirectory)/build"
     condition: and(eq(variables['Agent.OS'], 'Linux'), ne(variables['System.PullRequest.IsFork'], 'True'))
 
   - task: PublishBuildArtifacts@1
-    displayName: "Publish artifacts: package"
+    displayName: "Publish artifacts: packages and vsix"
     inputs:
-      PathtoPublish: "$(build.artifactstagingdirectory)"
+      PathtoPublish: "$(build.artifactstagingdirectory)/build"
       ArtifactName: $(artifact_name)
     condition: and(eq(variables['Agent.OS'], 'Linux'), ne(variables['System.PullRequest.IsFork'], 'True'))
 
+  - task: CopyFiles@2
+    displayName: "Copy web to staging directory"
+    inputs:
+      Contents: |
+        dist/web/*.js*
```

Comment:
Does web also need readme? I'm guessing it doesn't need the NOTICE file or changelog.